core/src/main/resources/defaults/blueprints/cdp-opdb-702.bp

```diff
@@ -188,7 +213,8 @@
           "hbase-GATEWAY-BASE",
           "hbase-REGIONSERVER-BASE",
           "hdfs-DATANODE-BASE",
-          "hdfs-GATEWAY-BASE"
+          "hdfs-GATEWAY-BASE",
+          "phoenix-PHOENIX_QUERY_SERVER-BASE"
```

Comment:
If we are deploying N PQS instances, we'll also need to pull back the memory allocations for the RegionServer on the worker nodes. Do we explicitly set a heapsize for PQS via CM?

The other way forward would be to add just one PQS on the gateway host which has lots of free memory.

Both approaches have their pro's and con's. I don't feel strongly either way :)

Reply:
Neither do I. Having multiple PQS servers is really only useful if we have some kind of load balancing, but I hope to have that sorted out in the next few months.

We are using the default CM java settings for PQS, and do not set an explicit memory limit. I will look into the memory settings, and adjust them.